person_test.go

```diff
@@ -53,11 +53,19 @@ func TestLastName(t *testing.T) {
 func TestName(t *testing.T) {
 	p := New().Person()
 	name := p.Name()
+	gender := p.Gender()
```

Comment:
considering that you have a test specific for this method, can you remove the changes in this tests? I guess this is the last item